knack/help.py

```diff
@@ -464,8 +464,7 @@ def _layout_items(items):
                     'line_len': line_len
                 }
                 layout['summary'] = layout['summary'].replace('\n', ' ')
-                if line_len > self.max_line_len:
-                    self.max_line_len = line_len
+                self.max_line_len = max(line_len, self.max_line_len)
```

Comment:
To solve `pylint` error:

```
> pylint knack --rcfile=.pylintrc --reports n --disable I0013
************* Module knack.help
knack\help.py:467:16: R1731: Consider using 'max_line_len = max(max_line_len, line_len)' instead of unnecessary if block (consider-using-max-builtin)
knack\help.py:589:16: R1731: Consider using 'max_line_len = max(max_line_len, line_len)' instead of unnecessary if block (consider-using-max-builtin)
```